great_expectations/expectations/metrics/metric_provider.py

```diff
@@ -1,12 +1,14 @@
+import copy
```

Comment:
This is coming from develop but is also not used there

Reply:
Correct -- leaving as it was in develop.